### ISSUE TITLE
\TextField and \ChoiceMenu don't begin at the start of the line whene there is no text before

### ISSUE THREAD

**Issue by CarLaTeX** [latex3/hyperref#323]
Even if the \TextField and \ChoiceMenu are the first objects in a line with \noindent, they don't begin and the start of the line, there is a little gap.

See this post on TeX.SE: [Why don't \TextField and \ChoiceMenu begin at the start of the line?](https://tex.stackexchange.com/questions/489283/why-dont-textfield-and-choicemenu-begin-at-the-start-of-the-line).

```
\documentclass{article}
\usepackage{hyperref}
\usepackage{showframe}

\begin{document}
\begin{Form}[action=mailto:my_email@gmail.com?subject={The submitted form},encoding=html, method=post]
\noindent Why doesn't the following field begin at the start of the line?
\vspace{1ex}

\noindent\TextField[name=a,width=5cm,charsize=8pt, height=.5cm]{}
\vspace{4ex}

\noindent Adding \texttt{bordersep=0pt} doesn't help:
\vspace{1ex}

\noindent\TextField[bordersep=0pt,name=b,width=5cm, height=.5cm]{}
\vspace{4ex}

\noindent The same with a choice menu:
\vspace{1ex}

\noindent\ChoiceMenu[combo, name=choice, width=5cm]{}{Choice 1, Choice 2}
\end{Form}
\end{document}
```

**Comment by muzimuzhi** [comment]
Duplicate of #93, whose fix was shipped with [v7.00a](https://github.com/latex3/hyperref/releases/tag/release-7.00a), released in Oct 2019. (Thus the reported problem is not reproducible now.)

It seems you posted the same problem both to https://github.com/ho-tex/oberdiek and to current repository on May 6, 2019, while unconnected.

**Comment by CarLaTeX** [comment]
> Duplicate of #93, whose fix was shipped with [v7.00a](https://github.com/latex3/hyperref/releases/tag/release-7.00a), released in Oct 2019. (Thus the reported problem is not reproducible now.)
> 
> It seems you posted the same problem both to https://github.com/ho-tex/oberdiek and to current repository (was named https://github.com/ho-tex/hyperref/ then) on May 6, 2019, while unconnected.

Yes, I copied here because I was asked to. If it's fixed please close.

**Comment by CarLaTeX** [comment]
No problem at all :) Bye
